src/RpcServer/RpcServer.cs

```diff
@@ -129,11 +133,41 @@ public void StartRpcServer()
             }))
             .Configure(app =>
             {
+                if (settings.EnableCors)
+                    app.UseCors("All");
```

Comment:
the design looks good!

Reply:
Coincidentally, we got this problem while using gitpod to access RPC nodes.
I will test this PR asap.